### ISSUE TITLE
The step to get "twitter API" (week04)

### ISSUE THREAD

**Issue by iiiJenny** [hupili/python-for-data-and-media-communication-gitbook#45]
* Which chapter of book:week04

## Step 1
Go to https://apps.twitter.com/app/new and click 'Apply for a developer account'.
## Step 2
Choose “Personal use” & Enter your desired Application Name, Primary country (hk) and so on.
**_The most important is_** : Describe in your own words what you are buiding

![image](https://user-images.githubusercontent.com/42435100/46203467-6e48dd80-c34c-11e8-8f84-d30310b0d008.png)

you have to answer each question as detailed as possible. the following is an example:

**1. I’m using Twitter’s APIs to practice my data collection and analysis skills in the Big Data Analysis course. I am currently a postgraduate student in Hong Kong Baptist University, majoring in Communication, and normally collect users comment data for mass communication study research.
2. As for the methods and techniques I plan to conduct, here is our course GitHub open book. You can take it as reference. https://github.com/hupili/python-for-data-and-media-communication-gitbook/blob/master/notes-week-04.md#use-api-via-function-calls-to-other-modules-packages
3. I only use it to do data collection practice, will not use case to tweeting, retweeting or liking content.
4. Tweets will be displayed on our final project presentation for academic use.
5. Finally, I will comply with the Policies of twitter. 
I am looking forward to your favorable reply.**

![image](https://user-images.githubusercontent.com/42435100/46203968-1ca15280-c34e-11e8-8db7-d8fc00a60ddf.png)

and then, submit your application! ->verify your email

and then, your application may under review or! jump to a new page :
![image](https://user-images.githubusercontent.com/42435100/46204207-faf49b00-c34e-11e8-9fa8-336e5ddb5da1.png)

**Comment by hupili** [comment]
@ChicoXYC , please leave a pointer here once it is merged into FAQ. Then we can close the issue.

**Comment by ChicoXYC** [comment]
Update into the FAQs. Please refer here: https://github.com/hupili/python-for-data-and-media-communication-gitbook/blob/a0a3c2c2287205f75e0bb4662dcafac745c53c24/module-python-twitter.md#how-to-apply-twitter-api-key
